app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,30 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
+    @user = current_user
+    @posts = @user.posts
   end
 
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @new_post = Post.new
+  end
+
+  def create
+    puts params
+    user = current_user
+    new_post = Post.new(post_params)
+    new_post.author = user
+    if new_post.save
+      redirect_to user_posts_url(id: user.id)
+    else
+      redirect_to new_post_url(user_id: user.id)
+    end
+  end
+
+  def post_params
+    params.require(:post).permit(:Title, :Text)
```

Comment:
- [ ] it would be better if you style your app according to the given wireframe in order to complete the requirements 
- wireframes
<img width="949" alt="Screenshot 2023-02-03 at 17 40 00" src="https://user-images.githubusercontent.com/61790587/216658426-aea002f2-8b4e-46f6-b8d8-7524f63b46e2.png">
- Yours 


<img width="279" alt="Screenshot 2023-02-03 at 17 35 34" src="https://user-images.githubusercontent.com/61790587/216658477-2db3993f-f507-4f27-9536-b095e5b9908f.png">
<img width="180" alt="Screenshot 2023-02-03 at 17 35 43" src="https://user-images.githubusercontent.com/61790587/216658483-a8f90166-d848-4496-bbc1-4d08a412cb72.png">